prow/job/github_auth.py

```
@@ -0,0 +1,138 @@
+REPO_RELEASE_TESTS = f"{OPENSHIFT_OWNER}/{RELEASE_TESTS_REPO}"
+
+
+def _github_app(app_id_env: str, key_env: str) -> GitHubApp:
+    app_id = os.environ.get(app_id_env)
+    private_key_path = os.environ.get(key_env)
+    if not app_id or not private_key_path:
+        print(f"Missing {app_id_env} or {key_env}, exit...")
+        sys.exit(1)
+    try:
+        return GitHubApp(app_id, private_key_path)
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)